src/plonemeeting/portal/core/utils.py

```
@@ -222,8 +246,12 @@ def get_base_api_url_for_meeting_items(institution, meeting_external_uid, item_e
     return url
 
 
-def get_api_url_for_presync_meeting_items(institution, meeting_external_uid, item_external_uids=[]):
-    url = get_base_api_url_for_meeting_items(institution, meeting_external_uid, item_external_uids)
+def get_api_url_for_presync_meeting_items(
+    institution, meeting_external_uid, item_external_uids=[]
```

Ruff (0.15.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)